Dockerfile.dev

```diff
@@ -46,7 +46,7 @@ RUN curl --silent -o-  https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/inst
 RUN npm install -g yarn
 RUN yarn cache clean
 RUN yarn install --verbose
-RUN apt-get install chromium -y
+#RUN apt-get install chromium -y
```

Comment:
Comment.

Reply:
Ah yeah, let me revisit that.

Reply:
@tsmartt I've resolved this in my other PR. I had to do some other updates to get things working on the M1, can we just ignore this for now instead of rebasing/testing builds across different branches?